gradle/check.gradle

```diff
@@ -69,13 +69,14 @@ quickCheck.dependsOn checkstyleMandatory
 apply plugin: 'com.github.spotbugs'
 
 spotbugs {
-    toolVersion = '3.1.7'
+//  toolVersion = '4.2.1'
```

Comment:
Is the idea that this comment is documentation that the (default) `toolVersion` is 4.2.1?

Reply:
Yes, that was the idea. I could just take it out altogether.